services/src/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java

```diff
@@ -98,6 +100,24 @@ public void reviveSwitch(String switchId, String controllerAddress) {
                 "reviveSwitch operation for a-switch is not available on hardware env");
     }
 
+    @Override
+    public void stopController() {
```

Comment:
To keep our terminology, this method should be `stopSpeaker` or `stopFloodlight`.